packages/zoe/src/types.js

```diff
@@ -51,7 +51,9 @@
  * @property {MakeInstance} makeInstance
  * @property {Offer} offer
  * @property {(Instance) => Object} getPublicFacet
- *
+ * @property {IssuerKeywordRecord} getIssuers
```

Comment:
Should be

```js
@property {() => IssuerKeywordRecord} getIssuers
```
and likewise for the two below.

Reply:
Did you mean to put these three on ZoeService? All three look like they should be specific to a contract instance. Especially `getTerms`.

Reply:
It needs to be on the Zoe Service to be credible (the ERTP Payment pattern). We should have to pass in an `instance` (opaque object). That said, I think we do not want the instanceTable that was added in this commit.